### ISSUE TITLE
Pytorch spawner error

### ISSUE THREAD

**Issue by vfdev-5** [polyaxon/polyaxon#386]
### Describe the bug

I'm trying to start a distributed training using PyTorch
Experiment fails with the error:
```bash
Traceback (most recent call last): File "/polyaxon/polyaxon/scheduler/experiment_scheduler.py", line 493, in start_experiment use_sidecar=True) 
File "/polyaxon/polyaxon/scheduler/spawners/experiment_spawner.py", line 99, in __init__ self.job_uuids = self.create_job_uuids() 
File "/polyaxon/polyaxon/scheduler/spawners/pytorch_spawner.py", line 17, in create_job_uuids uuid.uuid4().hex for _ in self.get_n_pods(task_type=TaskType.WORKER)] 
TypeError: 'int' object is not iterable 
```

### To Reproduce

On a cluster with 3 nodes (CPU+GPU), I run 
```bash
polyaxon run -u -f config.yaml --name=distrib_training
```

```yaml
---
version: 1
kind: experiment

environment:
  node_selector:
    polyaxon: exp-gpu    
  resources:
      gpu:
        requests: 1
        limits: 1

  pytorch:
    n_workers: 2

    default_worker: 
      resources:
        gpu:
          request: 1
          limits: 1

build:
  image: private/polyaxon/pytorch:latest
  build_steps: 
  - pip install --no-cache --no-deps --upgrade git+https://github.com/pytorch/ignite.git

run:
  cmd:    
    - python -u main.py
```

### Environment
Polyaxon 0.4.0

**Comment by mouradmourafiq** [comment]
definitely a regression, there's also an (intentional) regression in the UI for showing logs of non master logs, both will will be solved in the 0.4.1.

**Comment by mouradmourafiq** [comment]
Closing this, a patch will be released in v0.4.1 today or tomorrow.
